### PR TITLE
persistent volume and persistent volume claim prom queries

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

**Comment by chancez** [comment]
LGTM can you squash?

File: charts/reporting-operator/templates/custom-resources/prom-queries/pod-persistent-volume-request.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportPrometheusQuery
+metadata:
+  name: "pod-persistent-volume-claim-request"
```
[the remark this answers is not in the document]
Comment (EmilyM1) [reply charts/reporting-operator/templates/custom-resources/prom-queries/pod-persistent-volume-request.yaml]: Solid point.
Fixed in 71cf57d306a09fea3d561b70b4963dd0a47ae0d3

File: charts/reporting-operator/values.yaml [hunk 2]
```diff
@@ -77,6 +77,10 @@ spec:
         spec:
           promsum:
             query: "pod-usage-memory-bytes"
+      pod-persistentvolume-request:
```
[the remark this answers is not in the document]
Comment (EmilyM1) [reply charts/reporting-operator/values.yaml]: Fixed in 71cf57d306a09fea3d561b70b4963dd0a47ae0d3

File: charts/reporting-operator/values.yaml [hunk 3]
```diff
@@ -77,6 +77,10 @@ spec:
...
+        spec:
+          promsum:
+            query: "pod-persistent-volume-claim-request"
```
[the remark this answers is not in the document]
Comment (EmilyM1) [reply charts/reporting-operator/values.yaml]: Fixed in 71cf57d306a09fea3d561b70b4963dd0a47ae0d3

**Comment by chancez** [comment]
LGTM. Squash & I'll approve.

**Pull request merged**